make0x.py

```diff
@@ -9,7 +9,11 @@
 def output_on_failure(cmd, cwd):
     print(f'$ cd {cwd} && {cmd}')
     ret = subprocess.run(
-        cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, cwd=cwd,
+        cmd,
+        cwd=cwd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        universal_newlines=True,
```

Comment:
Maybe like this? Using `universal_newlines` instead of `text` for better backwards compatibility (lol) :D